pelican/utils.py

```diff
@@ -646,22 +638,37 @@ def process_translations(content_list, order_by=None):
                 for x in lang_items:
                     logger.warning('\t%s', x.source_path)
 
+        # items with `translation` metadata will be used as translations...
+        candidate_items = list(filter(
+            lambda i:
+                i.metadata.get('translation', 'false').lower() == 'false',
+            items))
+        # ...unless all items with that slug are translations
+        if not candidate_items:
+            logger.warning('All times of "%s" are translations', slug)
+            candidate_items = items
+
         # find items with default language
-        default_lang_items = list(filter(
+        original_items = list(filter(
             attrgetter('in_default_lang'),
-            default_lang_items))
+            candidate_items))
+
+        # if there is no article with default language, go back one step
+        if not original_items:
+            original_items = candidate_items
 
-        # if there is no article with default language, take an other one
-        if not default_lang_items:
-            default_lang_items = items[:1]
+        # display warning if there are several base items
+        if len(original_items) > 1:
+            logger.warning('There are %s original (not translated) items "%s"',
+                           len(original_items), slug)
 
         if not slug:
             logger.warning(
```

Comment:
This happens in one of the folliwng cases:
- for that slug, there are several versions with all have the default lang and all not marked as translations
- for that slug, there are several versions where none have the default lang and all not marked as translations

So we have one warning saying that there are several originals items for that slug. If there are more than 2, I agree that the warning will still be there if they only change one.

I'm not sure why you wrote that the slug is empty?